tianshou/policy/modelfree/fqf.py

```diff
@@ -0,0 +1,157 @@
+from tianshou.utils.net.discrete import FractionProposalNetwork, FullQuantileFunction
+import torch
+import numpy as np
+import torch.nn.functional as F
+from typing import Any, Dict, Optional, Union
+
+from tianshou.data import Batch, to_numpy, ReplayBuffer
+from tianshou.policy import DQNPolicy, QRDQNPolicy
+
+
+class FQFPolicy(QRDQNPolicy):
+    """Implementation of Fully-parameterized Quantile Function. arXiv:1911.02140.
+
+    :param torch.nn.Module model: a model following the rules in
+        :class:`~tianshou.policy.BasePolicy`. (s -> logits)
+    :param torch.optim.Optimizer optim: a torch.optim for optimizing the model.
+    :param float discount_factor: in [0, 1].
+    :param int num_fractions: the number of fractions to use. Default to 32.
+    :param float ent_coef: the coefficient for entropy loss. Default to 0.
+    :param int estimation_step: the number of steps to look ahead. Default to 1.
+    :param int target_update_freq: the target network update frequency (0 if
+        you do not use the target network).
+    :param bool reward_normalization: normalize the reward to Normal(0, 1).
+        Default to False.
+
```

Comment:
what are `fraction_model` and `fraction_optim`?

Reply:
@nuance1979 ？

Reply:
Sorry for the late reply. Fixed.